index.html

```diff
@@ -1,53 +1,179 @@
+            </div>
+            <div class="lister">
+                <p><strong>CANOPY</strong></p>
+                <div class="point"></div>
+                <P>Back End Dev</P>
+                <div class="point"></div>
+                <p>2015</p>
+            </div>
+
+            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
+            </p>
+            <ul>
+                <li id="html"><a href="#"> html</a></li>
+                <li id="css"><a href="#">css</a></li>
+                <li id="javaScript"><a href="#">javaScript</a></li>
+            </ul>
+            <div class="to-project"><a href="#">See Project</a></div>
+        </div>
+        <div class="container container3">
+            <div class="Card1"><img src="images/img3.png" alt="" class="card1"></div>
+            <div>
+                <h1>Tonic</h1>
+            </div>
+            <div class="lister">
+                <p><strong>CANOPY</strong></p>
+                <div class="point"></div>
+                <P>Back End Dev</P>
+                <div class="point"></div>
+                <p>2015</p>
+            </div>
+
+            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
+            </p>
+            <ul>
+                <li id="html"><a href="#"> html</a></li>
+                <li id="css"><a href="#">css</a></li>
+                <li id="javaScript"><a href="#">javaScript</a></li>
+            </ul>
+            <div class="to-project"><a href="#">See Project</a></div>
+        </div>
+        <div class="container container4">
+            <div class="Card1"><img src="images/img4.png" alt="" class="card1"></div>
+            <div>
+                <h1>Multi-Post <br>Stories</h1>
+            </div>
+            <div class="lister">
+                <p><strong>CANOPY</strong></p>
+                <div class="point"></div>
+                <P>Back End Dev</P>
+                <div class="point"></div>
+                <p>2015</p>
+            </div>
+
+            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
+            </p>
+            <ul>
+                <li id="html"><a href="#"> html</a></li>
+                <li id="css"><a href="#">css</a></li>
+                <li id="javaScript"><a href="#">javaScript</a></li>
+            </ul>
+            <div class="to-project"><a href="#">See Project</a></div>
+        </div>
+        <section class="About-section">
+            <div class="about-title">About <br> Myself</div>
+            <p class="para2">Hello I'm a software developer I can help you build a product, feature or website
+                look through some of my work and experience! if you like what you see and havd a Project
+                you need coded, don't hesitate to contact me.
+                <div class="connect">LET'S CONNECT</div>
```

Comment:
- [ ] Kindly change the font-weight of the text to match the Figma design.
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (colors, font-weight) using the Figma Template provided. It would be nicer to change the color of `Let's connect` and the border color of `Get my resume button` to the colors that are provided in the Figma design.
- Your design:
<img width="219" alt="image" src="https://user-images.githubusercontent.com/77190203/154455471-74755837-ff73-4694-8bf0-cb0696e0f0f8.png">

- Figma design:
<img width="188" alt="image" src="https://user-images.githubusercontent.com/77190203/154455741-144425e3-cb06-4827-b6b8-dc006dcc2a79.png">